docs/document/docs/components/desktop/tabs.md

```diff
@@ -0,0 +1,128 @@
+| visible | boolean | true | Show/Hide the component | |
+| items | Array\<[Item](#item)\> | [] | List of tabs to display | Will result an error if the value of items is not an array |
+
+#### Item
+
+| Name   | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| content | string/HTMLElement | "" | Tab content | |
+| label | string | "" | Tab name | |
+| value | string | "" | Key of each tab<br>*`Required` and `Unique` | `value` is unique and required.<br>Will result an error if the `value` is duplicated in `items` or not specified |
+| disabled | boolean | false | Enable/Disable the tab | |
+| visible | boolean | true | Show/Hide the tab | |
+
+### Event
+
+Here is a list of events that can be specified:
+
+| Name | Type | Description | Remark |
+| :--- | :--- | :--- | :--- |
+| change | function | Event handler of when selected tab is changed |  It will pass the event object as the argument<br><br>You can receive the following values in event.detail<br><li>event.detail.oldValue : "value" before the change</li><li>event.detail.value : "value" after the change</li> |
+
+### Constructor
+
+Tabs(options)<br>
+Here is a list of available constructors:
+
+#### Parameter
+
+| Name | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| options | object | {} | Object that includes component properties | |
+
+---
+## Sample Code
+
+> Please check the [package installation](../../getting-started/quick-start.md#installation) method first.
+
+Here is a sample code when all parameters are specified:
+
+```javascript
+const Kuc = Kucs['1.x.x'];
+
+const space = kintone.app.record.getSpaceElement('space');
+
+const firstContent = document.createElement("div");
+  const textArea = new Kuc.TextArea({
+    label: "Fruit",
+    requiredIcon: true,
+    value: "Apple",
+    error: "Error",
+    visible: true,
+    disabled: false,
+    placeholder: "",
+  });
+  firstContent.appendChild(textArea);
+
+  const secondContent = document.createElement("div");
+  const timePicker = new Kuc.TimePicker({
+    value: "11:30"
+  });
+  secondContent.appendChild(timePicker);
+
+  const thirdContent = document.createElement("div");
+  thirdContent.innerText = "tab3_content"
+
+  const tabs = new Kuc.Tabs({
+  borderVisible: true,
+  className: 'kuc-tabs-class',
+  id: 'sample-id',
+  items: [
+    {
+      label: 'TextArea With Placeholder',
+      content: firstContent,
+      value: 'tab-textarea',
+      disabled: false,
+    },
+    {
+      label: 'TimePicker',
+      content: secondContent,
+      value: 'tab-time-picker',
+      disabled: false,
+    },
+    {
+      value: 'tab-string-pattern',
+      label: 'Just String Pattern',
+      content: thirdContent,
+      disabled: false,
+    },
+  ],
+  value: 'tab-textarea',
+  visible: true,
+});
+space.appendChild(tabs);
+
+tabs.addEventListener('change', event => {
+  console.log(event);
+});
```

Comment:
How about improving like this? (I haven't tried behavior, so need checking)
- Fixed indent mistake
- Deleted unnecessary div
- Changed tab label, content

If ok, please update component preview, too.

```suggestion
const Kuc = Kucs['1.x.x'];

const space = kintone.app.record.getSpaceElement('space');

// Create each Tab content
const textArea = new Kuc.TextArea({
  label: "TextArea",
  requiredIcon: true,
  value: "This is sample.",
});

const timePicker = new Kuc.TimePicker({
  label: "Time"
  value: "11:30"
});

const text = "This is sample."

const tabs = new Kuc.Tabs({
  items: [
    {
      label: 'A',
      content: textArea,
      value: 'a',
      disabled: false,
    },
    {
      label: 'B',
      content: timePicker,
      value: 'b',
      disabled: false,
    },
    {
      label: 'C',
      content: text,
      value: 'c',
      disabled: false,
    },
  ],
  value: 'a',
  className: 'options-class',
  id: 'options-id',
  visible: true,
  borderVisible: true,
});
space.appendChild(tabs);

tabs.addEventListener('change', event => {
  console.log(event);
});
```

Reply:
Please copy and paste my translated_doc sample code.
It still has differences.